front-end/adminPanel.py

```diff
@@ -0,0 +1,160 @@
+import tkinter as tk
+from tkinter import ttk, messagebox, simpledialog
+from requestHandler import RequestHandler
+
+class AdminPanel(tk.Frame):
+    def __init__(self, master, manager, request_handler):
+        super().__init__(master)
+        self.manager = manager
+        self.request_handler = request_handler
+        self.grid_rowconfigure(0, weight=0)
+        self.grid_rowconfigure(1, weight=1) 
+        self.grid_columnconfigure(0, weight=1)
+        self.create_navbar()
+        self.create_layout()
+
+    def create_navbar(self):
+        self.navbar_frame = tk.Frame(self, bg="lightgray")
+        self.navbar_frame.grid(row=0, column=0, sticky="ew")
+        self.navbar_frame.grid_columnconfigure(0, weight=1)  # Rozciągnięcie na całą szerokość
```

Comment:
Widzę że lenistwo bierze górę i nawet komentarze z chata zostają ;)

Reply:
akurat to jest skopiowane z windowMenagera a komentarz został ponieważ jak znowu będę miał problem ze skalowaniem na cały ekran to będę wiedział co ostatnio przeoczyłem